docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md

```diff
@@ -44,51 +44,52 @@ NuGet packages can be installed into a .NET project of some kind. For this walkt
     dotnet add package Newtonsoft.Json
     ```
 
-2. After the command completes, open the `.csproj` file to see the added reference:
+2. After the command completes, open the *nuget1.csproj* file in Visual Studio to see the added NuGet package reference:
 
     ```xml
     <ItemGroup>
-      <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
+      <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     </ItemGroup>
     ```
 
 ## Use the Newtonsoft.Json API in the app
 
-1. Open the `Program.cs` file and add the following line at the top of the file:
+1. In Visual Studio, open the *Program.cs* file and add the following lines at the top of the file:
 
     ```cs
+    using nuget1;
     using Newtonsoft.Json;
     ```
 
-1. Add the following code before the `class Program` line:
+1. Add the following code to replace the `Console.WriteLine("Hello, World!");` statement:
 
     ```cs
-    public class Account
+    Account account = new Account
     {
-        public string Name { get; set; }
-        public string Email { get; set; }
-        public DateTime DOB { get; set; }
-    }
+        Name = "John Doe",
+        Email = "john@nuget.org",
+        DOB = new DateTime(1980, 2, 20, 0, 0, 0, DateTimeKind.Utc),
+    };
+    
+    string json = JsonConvert.SerializeObject(account, Formatting.Indented);
+    Console.WriteLine(json);
     ```
 
-1. Replace the `Main` function with the following:
+1. Add the following class definition after the `Console.WriteLine(json);` line:
 
     ```cs
-    static void Main(string[] args)
+    namespace nuget1
```

Comment:
Probably want to omit this namespace.

Reply:
@JonDouglas - From all I have learned, the namespace declaration is necessary for the new template. If you do not declare the namespace the build fails. @nkolev92 @zivkan - Can you comment if this is incorrect and let me know how to arrange the code so the namespace declaration is unnecessary?

Reply:
The namespaces are top level, so if you're recommending the class be added in the same file, then you likely don't need the namespace as it would've been declared top level.

Reply:
@nkolev92 @zivkan - The namespace has to be declared somewhere. In the old style template it was at the top of the file but in the new style it's supposed to be implied. It seems awkward partway through so I made a change to have them basically replacing the whole contents (one line) of the file rather than doing it in steps. This seems more straightforward. Let me know what you think.

Reply:
Seems fine by me. 
Sometimes our quickstarts get complicated because we try to follow a bunch of simple steps and we end up having to explain the steps in more detail that its needed. 

I like the simplification.